### ISSUE TITLE
aria2 1.24.0 build failure under OS X (MacPorts)

### ISSUE THREAD

**Issue by Schamschula** [aria2/aria2#683]
When building aria2 1.24.0 on Yosemite under MacPorts, I get the following error:

```
CXX      DHTMessageFactoryImpl.lo
DHTGetPeersMessage.cc:94:48: error: implicit instantiation of undefined template 'std::__1::array<unsigned char, 16>'
  std::array<uint8_t, sizeof(struct in6_addr)> dst;
                                               ^
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/__tuple:95:65: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TYPE_VIS_ONLY array;
                                                                ^
1 error generated.
```

**Comment by tatsuhiro-t** [comment]
Thank you.  Fix committed via 1e59e35

**Comment by nmaier** [comment]
Oh yeah, my good old friend, the missing `#inlcude <array>`. Had to do the same change when building the 1.24.0 release.
Works for me on master after your commit, so closing.
